test/sqllogictest/int2vector.slt

```diff
@@ -0,0 +1,106 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# 🔬🔬 int2vector
```

Comment:
Might be a good idea to have some of the tests output a `int2vector` type, or is there a limitation that requires us to cast to `text` at the end?

Reply:
Yeah, because it doesn't support a binary encoding (yet), we have to output it as text to get it to work with testdrive. Definitely doable, just not a priority to get the type in Materialize.

Reply:
got it, that makes sense.